token-client/src/test/java/com/sap/cloud/security/xsuaa/http/HttpHeadersTest.java

```diff
@@ -38,9 +38,13 @@ public void equals_differentHeaders_isNotEqual() {
 		HttpHeaders headers2 = new HttpHeaders(anotherHeader);
 		HttpHeaders headers3 = new HttpHeaders(sameKeyDifferentValue);
 
-		assertThat(headers1).isNotEqualTo(headers2);
-		assertThat(headers1).isNotEqualTo(headers3);
+		assertThat(headers1).isNotEqualTo(headers2).isNotEqualTo(headers3);
 		assertThat(headers2).isNotEqualTo(headers3);
 	}
 
+	@Test
+	public void toStringTest() {
```

Comment:
pls just remove "Test" in test case name :-)

Reply:
@nenaraab it's clashing then with the super toString method